jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -146,12 +146,12 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
     [build non-compressedrefs vm (large heap)])])
 
   OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU($build_cpu)
-  if test "x$with_noncompressedrefs" = x; then
-    OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}_cmprssptrs"
-    OPENJ9_LIBS_SUBDIR=compressedrefs
-  else
+  if test "${with_noncompressedrefs+set}" = set || test "x$OPENJDK_TARGET_CPU_BITS" = x32; then
     OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}"
     OPENJ9_LIBS_SUBDIR=default
+  elif test "x$with_noncompressedrefs" = x; then
```

Comment:
seems this test is obsolete now and this could use `else`

Reply:
updated custom-hook.m4  and generated-configure.sh was regenerated via common/autoconf/autogen.sh